src/index.js

```diff
@@ -0,0 +1,144 @@
+const GARDENSKIES = {
+  sunny: '☁️ ☁️ ☁️ ☀️ ☁️ ☁️',
+  cloudy: '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️',
+  rainy: '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧',
+  snowy: '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨',
+};
+
+const CONDITIONS = {
+  hot: { landscape: '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂', text: 'red' },
+  warm: { landscape: '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷', text: 'orange' },
+  moderate: { landscape: '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃', text: 'yellow' },
+  chilly: { landscape: '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲', text: 'green' },
+  cold: { landscape: '❄️🌲⛄️🌲❄️🏂⛄️🌲❄️⛷🌲❄️🌲', text: 'teal' },
+};
```

Comment:
Style: I it would have been clearer if you called the key `color` instead `text`:

```suggestion
const CONDITIONS = {
  hot: { landscape: '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂', color: 'red' },
  warm: { landscape: '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷', color: 'orange' },
  moderate: { landscape: '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃', color: 'yellow' },
  chilly: { landscape: '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲', color: 'green' },
  cold: { landscape: '❄️🌲⛄️🌲❄️🏂⛄️🌲❄️⛷🌲❄️🌲', color: 'teal' },
};
```